www/source/demo/windows/steps/8.html.slim

```diff
@@ -0,0 +1,106 @@
+
+ blockquote
+  p <b>Note</b> The command above is for local testing only. If you have uploaded your package to Builder, you can export it by calling <code>hab pkg export docker origin/package</code>. The default is to use the latest stable release; however, you can override that by specifying a different channel in an optional flag.
+
+ p The following example shows how to start the Docker container running your Habitat package.
+
+ = code(:shell) do
+   | PS C:\contosouniversity> docker run -it myorigin/contosouniversity
+
+ p For more information on using the Habitat Studio, see #{link_to 'Plan Builds', '/docs/developing-packages#plan-builds'}.
+
+ h2 <a name="debug-plan">Debugging your plan</a>
+
+ p When debugging your plan, it is important to understand what you can use to debug buildtime issues versus what you can use to debug runtime issues.
+
+ h3 Debugging at buildtime
+
+ p If you have to customize how your web application is built, such as overriding build phases through callbacks, it is helpful to know what environment variables are available to you, to make sure you have successfully copied/referenced files, or performed the business logic you intended to at that specific point in the build.
+
+ p There are several #{link_to 'built-in functions','/docs/reference#utility-functions'} you can call in your plan.ps1 to help you with these issues, but you will likely find Powershell's own <code>#{link_to 'Set-PSBreakpoint','https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/set-psbreakpoint?view=powershell-6'}</code> to be the most helpful. You can use its <code>-Command</code> argument and set it to any of the build phase callback functions. When that function is invoked, you will be dropped into an interactive prompt where you can examine variables and files to debug your build.
+
+ h3 Debugging at runtime
+
+ p Debugging at runtime involves using a few different techniques to understand how your running service is behaving, how it interacts with peers in different topologies, how it handles configuration updates, and so on.
+
+ p For verifying that files were copied into the appropriate paths for your service and paths resolve correctly in runtime hooks, log into the VM or connect to the Docker container that is running the service and examine the files and directories at <code>/hab/svc/contosouniversity/</code>. You can access the file structure by running the following command.
+
+ = code(:shell) do
+   | PS C:\contosouniversity> docker exec -it CONTAINERID powershell
+
+ p Once in the container, to observe HTTP endpoint data (such as census or butterfly information), start up your service, then connect to the HTTP endpoint using <code>Invoke-WebRequest</code>. You can pipe the JSON output to <code>ConvertFrom-Json</code> and then <code>ConvertTo-Json</code> to provide a filtered and readable view on the output.
```

Comment:
Do we talk about "butterfly" anywhere else? Just wondering whether readers will understand what this refers to.